### DISCUSSION TITLE
Taskbar height

### DISCUSSION THREAD

**Discussion by Rimole** [valinet/ExplorerPatcher#2065] [Q&A]
https://github.com/valinet/ExplorerPatcher/discussions/830
tells how to change between one-line-display and two-lines-display height (or even more than two). 

When I do Right click on Taskbar, Tools/Symbols Bar, Address, that field is higher than one line and smaller than two lines. If the Taskbar is one line heigh, it automatically increases its height to that size, maybe 1.3 lines.

I do not want the Address Bar, but I want the increased height of the TaskBar (because then the clock also shows the weekday).

Where/how can I change the height of the TaskBar to a value between 1 and 2 lines, please? 

![taskbar](https://github.com/valinet/ExplorerPatcher/assets/34980325/a13649e8-41cc-4578-8812-4679e87a9551)

**Comment by pyrates999** [comment]
In windows 10, right click on the taskbar and untick lock the taskbar.  Then you can adjust its height between one or two rows or more.  Then right click on the taskbar and tick lock the taskbar.

This isn't something for EP as windows already offers this setting.

**Reply by Rimole** [reply]
"one or two rows" is exactly what I do _**not**_ want. I want the height which is automatically set, when I use the Address bar. But I do not want the Address bar, I want just the higher taskbar (but not so high as two rows). There is either somewhere a setting for the number of rows, which should be set to, say, 1.3, or I need another symbol tool bar as high as address bar but only 1px wide.

**Reply by pyrates999** [reply]
I've never heard of adjusting the height of the taskbar before.

The closest setting I can find is in the EP properties application, go to taskbar and change the icon size to small or large.  Then the taskbar height will be changed based on what you set it to.
